seed/challenges/01-front-end-development-certification/jquery.json

```diff
@@ -590,7 +590,16 @@
         "assert.isFalse((/<i>/gi).test($(\"h3\").html()), 'message: Do not alter any other text.');"
       ],
       "type": "waypoint",
-      "challengeType": 0
+      "challengeType": 0,
+      "nameEs": "Cambia el texto de un elemento usando jQuery",
+      "descriptionEs2": [
+        "Con jQuery, puedes cambiar el texto que está entre las etiquetas de apertura y cierre de un elemento.  Incluso puedes cambiar marcado HTML.",
+        "jQuery tiene una función llamada <code>.html()</code> que te permite añadir etiquetas HTML y texto dentro de un elemento.  Cualquier contenido que estuviese previamente dentro del elemento será remplazado por completo con el contenido que proveas usando esta función.",
+        "He aquí como reescribes y pones en itálicas el texto de nuestro encabezado:",
+        "<code>$(\"h3\").html(\"&#60;i&#62;jQuery Playground&#60;/i&#62;\");</code>",
+        "jQuery también tiene una función similar llamada <code>.text()</code> que sólo altera el texto sin añadir etiquetas.",
+        "Cambia el botón con identificación <code>target4</code> para dejar su texto en itálicas."
```

Comment:
`de tal forma que su texto esté en itálicas` en vez de `para dejar su texto en itálicas`. Creo que se entiende mejor de esa forma.